### PR TITLE
Add config info to readme

### LINKED ISSUE #17: More instructions for the config
The last time I tried it I don't remember it creating a config somewhere, so how does that actually work? 
What is the syntax and contents for the config? 
Is there an automatic way to generate a config so I can just fill it?

### PR DESCRIPTION
**Pull request by Madis0**
Fixes #17 
though I glad you added the comments inside the config already

### CODE REVIEW

**Comment by RozeFound** [comment]
Actually I found that config is not that optional since it have one required field `version`

**Comment by RozeFound** [comment]
Probably need to make prompt for it right in script

**Comment by Madis0** [comment]
In that case yeah, make it prompt and generate the config with missing mods.

**Comment by RozeFound** [comment]
I mean, prompt only for version

**Comment by Madis0** [comment]
Yes, it can prompt for version but it will then generate the config with that version and that config will have the resources it did not find.

**Comment by RozeFound** [comment]
Why generating config? it's counter-intuitive

**Comment by Madis0** [comment]
Okay, you can ask for just the version as well.

**Comment by RozeFound** [comment]
What about documenting new key? :D

File: README.md [hunk 1]
```diff
@@ -70,6 +70,29 @@ mmc-export [-h] [-c CONFIG] -i INPUT -f FORMAT [-o OUTPUT] [--github-auth GITHUB
...
+> No config entry found for resource: ModName
+
+Then you should add an entry to the end of the config like so:
+
+```
+[[Resource]]
+name = "ModName"
+filename = "the_name_of_the.jar" 
+url = "https://cdn.modrinth.com/data/abcdefg/versions/1.0.0/the_name_of_the.jar"
+action = "override"
```
Comment (RozeFound) [inline README.md:86]: But override will... Override. To apply config to resource, you must not specify action at all

**Pull request merged**